tests/test_data_types.py

```diff
@@ -455,79 +457,82 @@ def test_video_path_invalid(runner):
+    mol = wandb.Molecule.from_smiles("CC(=O)Nc1ccc(O)cc1")
+    mol.bind_to_run(mocked_run, "rad", "summary")
+    wandb.Molecule.seq_to_json([mol], mocked_run, "rad", "summary")
 
-        assert os.path.exists(mol._path)
+    assert os.path.exists(mol._path)
+    wandb.finish()
 
 
-def test_molecule_from_invalid_smiles(runner, mocked_run):
+def test_molecule_from_invalid_smiles(mocked_run):
     """Ensures that wandb.Molecule.from_smiles errs if passed an invalid SMILES string"""
     with pytest.raises(ValueError):
         wandb.Molecule.from_smiles("TEST")
+    wandb.finish()
 
 
-def test_molecule_from_rdkit_mol_object(runner, mocked_run):
+def test_molecule_from_rdkit_mol_object(mocked_run):
     """Ensures that wandb.Molecule.from_rdkit supports rdkit.Chem.rdchem.Mol objects"""
-    with runner.isolated_filesystem():
-        mol = wandb.Molecule.from_rdkit(rdkit.Chem.MolFromSmiles("CC(=O)Nc1ccc(O)cc1"))
-        mol.bind_to_run(mocked_run, "rad", "summary")
-        wandb.Molecule.seq_to_json([mol], mocked_run, "rad", "summary")
+    mol = wandb.Molecule.from_rdkit(rdkit.Chem.MolFromSmiles("CC(=O)Nc1ccc(O)cc1"))
+    mol.bind_to_run(mocked_run, "rad", "summary")
+    wandb.Molecule.seq_to_json([mol], mocked_run, "rad", "summary")
 
-        assert os.path.exists(mol._path)
+    assert os.path.exists(mol._path)
+    wandb.finish()
 
 
-def test_molecule_from_rdkit_mol_file(runner, mocked_run):
+def test_molecule_from_rdkit_mol_file(mocked_run):
     """Ensures that wandb.Molecule.from_rdkit supports .mol files"""
-    with runner.isolated_filesystem():
-        substance = rdkit.Chem.MolFromSmiles("CC(=O)Nc1ccc(O)cc1")
-        mol_file_name = "test.mol"
-        rdkit.Chem.rdmolfiles.MolToMolFile(substance, mol_file_name)
-        mol = wandb.Molecule.from_rdkit(mol_file_name)
-        mol.bind_to_run(mocked_run, "rad", "summary")
-        wandb.Molecule.seq_to_json([mol], mocked_run, "rad", "summary")
+    substance = rdkit.Chem.MolFromSmiles("CC(=O)Nc1ccc(O)cc1")
+    mol_file_name = "test.mol"
+    rdkit.Chem.rdmolfiles.MolToMolFile(substance, mol_file_name)
+    mol = wandb.Molecule.from_rdkit(mol_file_name)
+    mol.bind_to_run(mocked_run, "rad", "summary")
+    wandb.Molecule.seq_to_json([mol], mocked_run, "rad", "summary")
 
-        assert os.path.exists(mol._path)
+    assert os.path.exists(mol._path)
+    wandb.finish()
 
 
-def test_molecule_from_rdkit_invalid_input(runner, mocked_run):
+def test_molecule_from_rdkit_invalid_input(mocked_run):
     """Ensures that wandb.Molecule.from_rdkit errs on invalid input"""
     mol_file_name = "test"
     with pytest.raises(ValueError):
         wandb.Molecule.from_rdkit(mol_file_name)
+    wandb.finish()
 
 
 def test_html_str(mocked_run):
     html = wandb.Html("<html><body><h1>Hello</h1></body></html>")
     html.bind_to_run(mocked_run, "rad", "summary")
     wandb.Html.seq_to_json([html], mocked_run, "rad", "summary")
     assert os.path.exists(html._path)
+    wandb.finish()
```

Comment:
I'm curious why we added all of these?  The `mocked_run` object should be calling finish after yielding it to the test.